### ISSUE TITLE
RuntimeError: Cannot load library 'libov_intel_myriad_plugin.so: dlopen(libov_intel_myriad_plugin.so

### ISSUE THREAD

**Issue by jdc711** [openvinotoolkit/openvino#10499]
Hi, I am trying to run benchmark_app on the NCS2 MYRIAD stick, however, it says that it cannot load this library. Here is the error:
dongwoo@Joshuas-MacBook-Pro-3 build % benchmark_app -m yolov4.xml -d MYRIAD
[Step 1/11] Parsing and validating input arguments
[ WARNING ]  -nstreams default value is determined automatically for a device. Although the automatic selection usually provides a reasonable performance, but it still may be non-optimal for some cases, for more information look at README. 
[Step 2/11] Loading OpenVINO
[ ERROR ] Cannot load library 'libov_intel_myriad_plugin.so: dlopen(libov_intel_myriad_plugin.so, 2): image not found
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/openvino/tools/benchmark/main.py", line 85, in run
    supported_config_keys = benchmark.core.get_metric(device, 'SUPPORTED_CONFIG_KEYS')
RuntimeError: Cannot load library 'libov_intel_myriad_plugin.so: dlopen(libov_intel_myriad_plugin.so, 2): image not found

I am using a macbook pro and followed the build for the macos. My specs are as follows:
Mac OS Version: 11.6.2
Python Version: 3.7.12
CMake Version: 3.21.4
Clang Version: Apple clang version 13.0.0 (clang-1300.0.29.30)

**Comment by brmarkus** [comment]
Which version of OpenVINO are you using?
Can you summarize how you built and installed it, please?

Have you "sourced the environment", like "source [my/installed/folder/]bin/setupvars.sh"?

For a consistency-check - (not sure you are using an Intel-SOC based MacbookPro) - can you use "-d CPU" and/or "-d GPU" successfully, just to check OpenVINO has been setup properly?

Can you search and find the mentioned library "libov_intel_myriad_plugin.so"? Under Linux with something like "find / -name libov_intel_myriad_plugin.so"?

**Comment by jdc711** [comment]
Hi. I did not source the environment, so I tried doiing "source scripts/setupvars/setupvars.sh" , however, i got an error: setupvars.sh] ERROR: Unsupported Python version. Please install one of Python 3.6 - 3.9 (64-bit) from https://www.python.org/downloads/

However, I am using python3.7, and when I type 'which python3', it gives me path to 3.7.

**Comment by jdc711** [comment]
I followed to instructiions on buildforMacos on the wiki openvino.

**Comment by jdc711** [comment]
I am stil getting the errors above. I believe they have somethinig to do with how my python libraries are installed. 
Also, For the cmake command, I dont know if I am doing this correctly. I am following the  build for macos instructions (https://github.com/openvinotoolkit/openvino/wiki/BuildingForMacOS#software-requirements). I am using the commands below to find where the dylib and include_dir are located.
'find /usr/ -name 'libpython*m.dylib'
'find /usr/ -type d -name python3.7m'

However, each of these commands output a list of directories, so I am not sure which one to use for the cmake command
/usr//local/mysql-shell-8.0.19-macos10.15-x86-64bit/lib/mysqlsh/libpython3.7m.dylib
/usr//local/Cellar/python@3.7/3.7.12_1/Frameworks/Python.framework/Versions/3.7/lib/libpython3.7m.dylib
/usr//local/Cellar/python@3.7/3.7.12_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/config-3.7m-darwin/libpython3.7m.dylib

/usr//local/mysql-shell-8.0.19-macos10.15-x86-64bit/lib/mysqlsh/include/python3.7m
/usr//local/Cellar/python@3.7/3.7.12_1/Frameworks/Python.framework/Versions/3.7/include/python3.7m


Do you have any other suggestions on how I can build openvino for macos from scratch to succesfully use the MYRIAD NCS2 Stick

**Comment by jdc711** [comment]
To clarify, are the steps listed below all i need to do in order to run benchmark_app ? (with the cmake command replaced with my directories of course)

Build Steps

Clone submodules:
  cd openvino
  git submodule update --init --recursive
Create a build folder:
  mkdir build && cd build
Inference Engine uses a CMake-based build system. In the created build directory, run cmake to fetch project dependencies and create Unix makefiles, then run make to build the project:
  cmake -DCMAKE_BUILD_TYPE=Release ..
  make --jobs=$(nproc --all)

**Comment by jgespino** [comment]
@jdc711 Since you are using MYRIAD (NCS 2) you need to build OpenVINO from source as you mentioned. The location of python depends how you installed it. I installed python3.7 using `brew` and used the following commands.

Install Python3.7 using brew:
```
brew install python@3.7
echo 'export PATH="/usr/local/opt/python@3.7/bin:$PATH"' >> ~/.zshrc
```

cmake command:
```
cmake -DCMAKE_BUILD_TYPE=Release -DENABLE_PYTHON=ON -DPYTHON_EXECUTABLE=/usr/local/Cellar/python\@3.7/3.7.12_1/Frameworks/Python.framework/Versions/3.7/bin/python3.7m -DPYTHON_LIBRARY=/usr/local/Cellar/python\@3.7/3.7.12_1/Frameworks/Python.framework/Versions/3.7/lib/libpython3.7m.dylib -DPYTHON_INCLUDE_DIR=/usr/local/Cellar/python\@3.7/3.7.12_1/Frameworks/Python.framework/Versions/3.7/include/python3.7m ..
```

Please give it a try and let me know if you run into any issues.

Regards,
Jesus
